src/mal/sqlbackend/sql_statement.c

```diff
@@ -124,15 +124,32 @@ stmt_key(stmt *s)
 }
 
 /* #TODO make proper traversal operations */
+#ifdef HAVE_EMBEDDED
 stmt *
-stmt_atom_string(backend *be, const char *S)
+stmt_atom_string(backend *be, const char *S, int escape)
+{
+	const char *s = NULL;
```

Comment:
again, changes from default??

Reply:
Mine. The CSV export was still having problems on Windows. The file path was getting manipulated by the sql2str call, thus failing the export.